### PR TITLE
fixes to the debian packaging

### PR DESCRIPTION
**Pull request by obilodeau**
@fdurand please review commit per commit and once you are ok with the changes, we'll send this into integration.

bug covered:
- [#1465: captive portal show odd strings](http://packetfence.org/bugs/view.php?id=1465)
- [#1511: SSH and Telnet deAuth doesn't work on debian](http://packetfence.org/bugs/view.php?id=1511)

Unfortunately current integration is also still broken because of fix/pfcmd-suid so we will have to fix that first.

```
NEWS

Enhancements
 * Debian packages improvements

Bug fixes
 * Debian: issues with translations (#1465)
 * Debian: SSH and Telnet based deauthentication (#1511)
```

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -10,47 +10,47 @@ Homepage: http://www.packetfence.org/
...
+ apache2, apache2.2-common, apache2-utils, libapache2-mod-proxy-html,
+ apache2-mpm-prefork,
+# php web admin
+ libapache2-mod-php5, php-pear, php5-mysql, php5-gd, make, php-log,
 # process management
  procps,
-# perl stuff
- libapache-htpasswd-perl, libbit-vector-perl, libtext-csv-perl, 
+# perl basic components
+ perl-suid, liblist-moreutils-perl, libwww-perl, libtry-tiny-perl
```
Comment (fdurand) [inline debian/control]: Missing comma

File: debian/control [hunk 2]
```diff
@@ -10,47 +10,47 @@ Homepage: http://www.packetfence.org/
...
- libtry-tiny-perl, snmp, snmptrapfmt, libmime-lite-perl,
+ libdatetime-format-dateparse-perl, libdbi-perl,
+ libnet-telnet-perl, libregexp-common-perl,
+ libreadonly-perl, libtemplate-perl, libterm-readkey-perl,
+ libuniversal-require-perl, libthread-serialize-perl,
  libnet-ldap-perl, libcrypt-generatepassword-perl, perl-doc, dhcp3-server,
  bind9, librrds-perl, libnetpacket-perl (>= 1.3), libcache-cache-perl, libcarp-perl,
  libiptables-libiptc-perl, libload-perl, libmime-lite-tt-perl,
- libnet-appliance-session-perl, libnet-cli-interact-perl,
+ libnet-appliance-session-perl (= 1.36), libnet-cli-interact-perl,
```
Comment (fdurand) [inline debian/control]: On the repo there is the libnet-appliance-session-perl version 2.111080-1 that work with packetfence

Comment (obilodeau) [inline debian/control]: You are correct, the API changed with version 3, my mistake. Fixing it.

File: debian/control (hunk 1 above)
Comment (obilodeau) [inline debian/control]: resolved

File: debian/control (hunk 2 above)
Comment (obilodeau) [inline debian/control]: resolved

**Comment by obilodeau** [comment]
Should fix [#1511: SSH and Telnet deAuth doesn't work on debian](http://packetfence.org/bugs/view.php?id=1511) too now. Updated NEWS entry in pull request to reflect that.

Ready for another round of review please.

**Comment by fdurand** [comment]
ok to merge

**Pull request merged**